configurationdemo/source/java/src/org/kaaproject/kaa/demo/configuration/ConfigurationDemo.java

```diff
@@ -22,69 +22,85 @@
 import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
 import org.kaaproject.kaa.client.configuration.base.ConfigurationListener;
 import org.kaaproject.kaa.client.configuration.base.SimpleConfigurationStorage;
+import org.kaaproject.kaa.schema.sample.Configuration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.BufferedReader;
 import java.io.IOException;
-import java.util.List;
+import java.io.InputStreamReader;
+
 
 /**
  * A demo application that shows how to use the Kaa configuration API.
+ *
+ * @author Maksym Liashenko
  */
 public class ConfigurationDemo {
+
     private static final Logger LOG = LoggerFactory.getLogger(ConfigurationDemo.class);
+
     private static KaaClient kaaClient;
 
     public static void main(String[] args) {
         LOG.info("Configuration demo started");
-        LOG.info("--= Press any key to exit =--");
 
-        // Create the Kaa desktop context for the application.
+        /*
+         * Create the Kaa desktop context for the application.
+         */
         DesktopKaaPlatformContext desktopKaaPlatformContext = new DesktopKaaPlatformContext();
 
-        // Create a Kaa client and add a listener which displays the Kaa client configuration 
-        // as soon as the Kaa client is started. 
+        /*
+         * Create a Kaa client and add a listener which displays the Kaa client
+         * configuration as soon as the Kaa client is started.
+         */
         kaaClient = Kaa.newClient(desktopKaaPlatformContext, new SimpleKaaClientStateListener() {
             @Override
             public void onStarted() {
                 super.onStarted();
-                displayConfiguration();
+                LOG.info("Getting sampling period value from default configuration");
```

Comment:
If we already have some saved configuration on the kaaClient start wee will get value from config storage, so I suggest to replace _default_ with _default/saved_